thirdeye/thirdeye-frontend/app/utils/anomaly.js

```diff
@@ -1,6 +1,68 @@
 import moment from 'moment';
 import { isPresent } from '@ember/utils';
 import _ from 'lodash';
+import {
```

Comment:
Just curious what's the rationale behind choosing to implement it as a utils vs a service?

Reply:
https://guides.emberjs.com/v3.0.0/applications/services/ explains the criteria for when to create a service. THere is no need to require shared state etc here.

Reply:
DONE